lib/vagrant-libvirt/action/connect_libvirt.rb

```diff
@@ -30,7 +30,7 @@ def call(env)
 
           # Setup command for retrieving IP address for newly created machine
           # with some MAC address. Get it from dnsmasq leases table
-          ip_command = %q[ find /var/lib/libvirt/dnsmasq/ /var/lib/misc/ -name '*leases' -exec grep $mac {} \; | cut -d' ' -f3 ]
+          ip_command = %q[ grep $mac /proc/net/arp | awk '{print $1}' ]
```

Comment:
In my testing we can remove the grep and just use awk with

`ip_command = %q[ awk "/$mac/ {print \$1}" /proc/net/arp ]`